### ISSUE TITLE
Trying to figure out how to use

### ISSUE THREAD

**Issue by benjamin-kline** [maxogden/usfs-fstopo#1]
I don't know how to use this code. I have experience with C# and python but would reallly really like to get at these maps. Can you please tell me how to run this?

**Comment by max-mapper** [comment]
you can access these same maps through caltopo.com as well

**Comment by benjamin-kline** [comment]
True, but I'm trying to skim everything for free. My goal is to have my own
copies that I can hold on to.

**Comment by benjamin-kline** [comment]
To clarify, I'm trying to download all of the USFS maps in total, not individually like how you can in Caltopo or on the USFS website.

**Comment by max-mapper** [comment]
The instructions are in the readme, you can look up some tutorials online for running node.js and npm scripts + cli tools if you are unfamiliar with those

**Comment by benjamin-kline** [comment]
Sweet that's what I needed to know. I was trying to figure out what
language/editor I needed to use. Thanks man!
